src/aiida_quantumespresso/workflows/xspectra/crystal.py

```diff
@@ -173,6 +173,19 @@ def define(cls, spec):
             help=('Input namespace to provide core wavefunction inputs for each element. Must follow the format: '
                    '``core_wfc_data__{symbol} = {node}``')
         )
+        spec.input_namespace(
+            'symmetry_data',
+            valid_type=(orm.Dict, orm.Int),
+            dynamic=True,
+            required=False,
+            help=(
+                'Input namespace to define equivalent sites and spacegroup number for the system. If defined, will '
+                + 'skip symmetry analysis and structure standardization. Use *only* if symmetry data are known'
+                + 'for certain. Requires ``spacegroup_number`` (Int) and ``equivalent_sites_data`` (Dict) to be'
+                + 'defined separately. All keys in `equivalent_sites_data` must be formatted as "site_<site_index>".'
+                + 'See docstring of `get_xspectra_structures` for more information about inputs.'
+            )
+        )
```

Comment:
Pretty sure the `+` signs are superfluous since all strings are encapsulated by `()`
```suggestion
            help=(
                'Input namespace to define equivalent sites and spacegroup number for the system. If defined, will '
                'skip symmetry analysis and structure standardization. Use *only* if symmetry data are known '
                'for certain. Requires ``spacegroup_number`` (Int) and ``equivalent_sites_data`` (Dict) to be '
                'defined separately. All keys in `equivalent_sites_data` must be formatted as "site_<site_index>". '
                'See docstring of `get_xspectra_structures` for more information about inputs.'
            )
        )
```